common/test/acceptance/tests/lms/test_lms_courseware_search.py

```diff
@@ -68,8 +68,10 @@ def _auto_auth(self, username, email, staff):
         Logout and login with given credentials.
         """
         LogoutPage(self.browser).visit()
-        AutoAuthPage(self.browser, username=username, email=email,
-                     course_id=self.course_id, staff=staff).visit()
+        login_page = AutoAuthPage(self.browser, username=username, email=email,
```

Comment:
It was, as you said, "try-this" kind of thing. Locally, these tests don't fail, so the only way I could recheck this is by adding it to the PR and seeing what the result would be on Jenkins.